workflow-templates/slash-command-tag.yaml

```
@@ -0,0 +1,48 @@
+name: Check for '/tag' and create new Release
+
+on: issue_comment
+
+jobs:
+  job1:
+    runs-on: ubuntu-latest
+    name: Create new Release and Automerge the PR
+    steps:
+      - name: Check to see if comment is on a PR
+        if: ${{ github.event.issue.pull_request }}
+        run: |
+          echo "Comment on PR #${{ github.event.issue.number }}"
+
+      - name: Find command in PR comments
+        uses: xt0rted/slash-command-action@v1
+        id: command
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          command: tag
+          reaction: "true"
+          reaction-type: "hooray"
+          allow-edits: "false"
+          permission-level: none
```

```suggestion
          repo-token: ${{ secrets.GITHUB_TOKEN }}
          command: tag
          reaction: true
          reaction-type: "hooray"
          allow-edits: false
          permission-level: write
```

- For boolean values in YAML, let's not have them in quotes. Unless this action itself requires them to be strings?
- Let's try setting the permission-level to ppl with write access

---

- I do think the repo we are using for this (xt0rted/slash-command-action@v1) does require them to be strings but I can do a test run without the quotes first if you want me to.
- For the permission level, I will change it but it also wouldn't complete the PR even if someone triggers the action if they themselves don't have the ability to commit the PR, so it's safe with 'none' there too afaik.

---

- Could you try using boolean values in dev-null. If this action _does_ require strings, can you add a comment explaining that the action requires it. We wouldn't want someone in the future to be confused
- Is that so? The token we're using for merging is `GITHUB_TOKEN` so that token would have access to merging even if the person who has read-only access to the repo triggers it, right?

---

- Looks like it worked without the ""
- And it still logs who started the action, even if it was indirectly, so it shouldn't be able to do anything beyond their permissions but I'll change it to "write" instead of "none"